### ISSUE TITLE
no-commented-out-code marks jsdoc

### ISSUE THREAD

**Issue by ekim98** [Glavin001/tslint-clean-code#60]
Jsdoc is getting marked as invalid if there is no @params or the return type isn't void.

```js
    /**
     * @return {void}
     */
    okayVoid(): void {
        return;
    }

    /**
     * @param {Object} items
     * @return {string}
     */
    okayParams(items: Object): string {
        return '';
    }

    /**
     * @return {Object}
     */
    notOkay(): Object {
        return {};
    }
```

**Comment by Glavin001** [comment]
Thanks for reporting this issue! Submitting a Pull Request with a failing test case would be great. Here is the applicable file for tests: https://github.com/Glavin001/tslint-clean-code/blob/master/src/tests/NoCommentedOutCodeRuleTests.ts
If you're feeling up to the challenge of fixing the problem, too, here is the applicable file for the rule itself: https://github.com/Glavin001/tslint-clean-code/blob/master/src/noCommentedOutCodeRule.ts

Thanks in advance for contributing!

**Comment by aleclarson** [comment]
Related: `// TODO: ` is also false-flagged

**Comment by cheeZery** [comment]
I'd like to take a shot at this! :-)
